harvard-lts/Harvard-Geospatial-Library
# Replace analytics with new google tag manager tracking id

**Replace analytics with new google tag manager tracking id**
* * *

# What does this Pull Request do?
Updates the analytics we are using to reference the new tracking id that is being applied across library systems.

# How should this be tested?
* Spin up container
* Inspect code to see that the new id (GTM-WSHSDQ5) and code are in the correct places (refer to [GTM installation instructions](https://tagmanager.google.com/#/admin/accounts/6006463004/containers/62544873/install?containerDraftId=21), this is the code that should be in the `<head>` and `<body>` parts of the page)
* Test in preview mode using tagmanager.google.com ([video ](https://drive.google.com/file/d/1wB3iJphfdrj28KLRgw456cnKeMNcqKb2/view?usp=sharing)for how you can check this locally *works best to test in Chrome)
* Once in production, we can test again to confirm that it's firing correctly on both the Google Tag Manager and Google Analytics side of things

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@phil-plencner-hl @enriquediaz
